src/elements/JobsPanelDataInputsTable/JobsPanelDataInputsTable.js

```diff
@@ -27,29 +27,55 @@ export const JobsPanelDataInputsTable = ({
   match,
   panelState
 }) => {
-  const pathScheme = inputsState.newInput.path.pathType
+  const jobsPanelRef = useRef(null)
+
+  const handleDocumentClick = useCallback(
+    event => {
+      if (!jobsPanelRef.current?.contains(event.target)) {
+        handleAddNewItem(true)
+        inputsDispatch({
+          type: inputsActions.SET_PATH_PLACEHOLDER,
+          payload: ''
+        })
+      }
+    },
+    [handleAddNewItem, inputsDispatch]
+  )
+
+  useEffect(() => {
+    if (jobsPanelRef.current) {
+      document.addEventListener('click', handleDocumentClick)
+
+      return () => {
+        document.removeEventListener('click', handleDocumentClick)
+      }
+    }
+  }, [handleDocumentClick, jobsPanelRef])
 
   return (
     <JobsPanelTable
       addNewItem={inputsState.addNewInput}
       className="data-inputs"
+      comboboxMatchesList={comboboxMatchesList}
```

Comment:
pass here prop "sectionData" wich will be the object, and set there comboboxMatchesList 

```suggestion
      sectionData={{comboboxMatchesList}}
```